src/Illuminate/Foundation/Console/BugCommand.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+namespace Illuminate\Foundation\Console;
+
+use Closure;
+use Illuminate\Console\Command;
+use Illuminate\Support\Composer;
+use Illuminate\Support\Facades\DB;
+use Illuminate\Support\Str;
+use Symfony\Component\Console\Attribute\AsCommand;
+
+#[AsCommand(name: 'bug')]
+class BugCommand extends Command
+{
+    /**
+     * The console command signature.
+     *
+     * @var string
+     */
+    protected $signature = 'bug';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Start to create a bug report';
+
+    /**
+     * Execute the console command.
+     *
+     * @return int
+     */
+    public function handle()
+    {
+        $osCommand = $this->detectOsName();
+        $url = $this->getUrl();
+
+        exec("$osCommand $url");
+
+        return 0;
+    }
+
+    /**
+     * Detect the OS name and get the current command.
+     *
+     * @return string
+     */
+    protected function detectOsName()
+    {
+        return match (PHP_OS_FAMILY) {
+            'Windows' => 'start',
+            'Linux', 'Darwin' => 'xdg-open',
+        };
+    }
+
+    /**
+     * Get the full url.
+     *
+     * @return string
+     */
+    protected function getUrl()
+    {
+        $dbInfo = config('database.default') . "-" . DB::connection()->getPdo()->getAttribute(\PDO::ATTR_SERVER_VERSION);
+
+        $url = "https://github.com/laravel/framework/issues/new?assignees=&labels=&projects=&template=Bug_report.yml";
+        $url .= "&laravel_version=" . $this->laravel->version();
+        $url .= "&php_version=" . phpversion();
+        $url .= "&database_info=" . $dbInfo;
+
+        return $url;
```

Comment:
Please [filter and validate](https://www.php.net/manual/en/filter.filters.validate.php#:~:text=compatible%20regular%20expression.-,FILTER_VALIDATE_URL,-%22validate_url%22) URL before you made `exec`
It is dangerous to pass something to `exec` which is not validated.

Reply:
Why need to validate url? the url is const and there isn't change!

Reply:
because you are dealing with exec function, it allows you to execute the code, so prevent data to b malformed, just in case

Reply:
I don't think there is a need to validate the URL, but if is needed Taylor says it